.eslintrc

```diff
@@ -9,6 +9,7 @@
     "no-param-reassign": 1,
     "prefer-destructuring": 1,
     "no-nested-ternary": 1,
+    "no-underscore-dangle": [ 2, { "allow": ["__dirname"], "allowAfterThis": true } ],
```

Comment:
👍 But better handled with `webpack-defaults`, since the `eslint` config is synced there, please additionally open a PR in `eslint-config-webpack` so the next `webpack-defaults` update is not overriding this